treeherder/config/settings.py

```diff
@@ -315,6 +315,7 @@
 FAILURE_LINES_CUTOFF = 35
 
 BZ_API_URL = "https://bugzilla.mozilla.org"
+BZ_API_KEY = env("BUGZILLA_API_KEY", default="")
```

Comment:
I'd use a default of `None`, and then make `create_bug()` return an error if it's not set.